opmd_viewer/openpmd_timeseries/interactive.py

```diff
@@ -173,13 +177,13 @@ def refresh_species(b):
                 dropdown_button.options = avail_records
 
             # Put back the previous value of the refreshing button
-            ptcl_refresh_button.value = saved_refresh_value
+            ptcl_refresh_toggle.value = saved_refresh_value
 
-        def change_t(name, value):
+        def change_t(change):
             "Plot the result at the required time"
-            self.current_t = 1.e-15 * value
-            refresh_field()
-            refresh_ptcl()
+            self.current_t = 1.e-15 * change['new']
+            refresh_field( change )
```

Comment:
you could keep `()` because forwarding the caller object might not help at this point (it is already handled)